descriptors/ModuleDescriptor-template.json

```diff
@@ -88,6 +88,14 @@
           "permissionsRequired": ["inventory.instances.item.post"],
           "modulePermissions": ["inventory-storage.instances.item.post",
                                 "inventory-storage.instances.item.get"]
+        }, {
+          "methods": ["POST"],
```

Comment:
As a new endpoint is being introduced, please increment the `inventory` interface minor version.